packages/frontend/app/routes/settings.tsx

```diff
@@ -10,6 +10,7 @@ import { PageHeader } from '~/components'
 import { Button, Input } from '../components/ui'
 import variables from '../lib/envConfig.server'
 import { redirectIfUnauthorizedAccess } from '../lib/kratos_checks.server'
+export const timeoutTwoPhase = BigInt(60)
```

Comment:
I think I need a little more context to understand why we're setting a static value for the timeoutTwoPhase variable. From what I'm reading in the comments we want the two phase to be optional, then surely this value should be something that can be set in the admin UI rather than using a constant value anyway?

But yes @mkurapov is right that the settings.tsx page is actually the frontend code for rendering the admin UI account settings. So we'd need a better spot for storing this value if it should be a static value. Maybe we can make a config file for it in the utils folder. 

With Remix there is a mix of frontend and server side code which is also better kept separate. The timeoutTwoPhase value actually only needs to be on the server side. So we could make a file called `packages/frontend/app/utils/config.server.ts` which will keep that information on the server then (all our requests are sent from the server).

Reply:
Decided to add an input field for the timeout value on two-phase commits. 
See this draft PR for progress [https://github.com/interledger/rafiki/pull/2721](https://github.com/interledger/rafiki/pull/2721)

Reply:
@mkurapov is this true for all the changes that affect `frontend`:

- [ ] withdrawAssetLiquidity
- [ ] createIncomingPaymentWithdrawal
- [ ] createOutgoingPaymentWithdrawal
- [ ] withdrawPeerLiquidity
- [ ] createWalletAddressWithdrawal

Reply:
> Surely the Admin UI is just a face for the Admin APIs

I think these particular liquidity mutations are a bit of an anomaly, since they are almost always meant to be called automatically by the ASE backend only. I just don't know of a scenario where we will want to create a "pending" withdrawal via the admin UI that will end up expiring. If the ASE wants to do reconciliation it just seems like they would either want to do it, or not, without an "in-between" pending transfer state, if that makes sense

Reply:
Thank you @JoblersTune and @mkurapov. I have updated this PR to reflect @JoblersTune's suggestion (`config.server.ts`) and also include an optional environment variable for `WITHDRAWAL_TIMEOUT_SECONDS`.

> I am going to resolve this thread for now, but I would love to be part of a conversation where we cover the touch points for each of the withdrawal types, when it would make sense to use 2-phase and when not...